packages/mint-components/src/components/sqm-task-card/progress-bar/progress-bar-view.tsx

```diff
@@ -0,0 +1,546 @@
+      marginRight: "var(--sl-spacing-x-small)",
+      fontSize: "var(--sl-font-size-x-small)",
+      display: "grid",
+      lineHeight: "45px",
+      userSelect: "none",
+    },
+    "& .progress-bar.repeatable-steps": {
+      marginLeft: "var(--sl-spacing-x-small)",
+    },
+    "&[data-steps]": {
+      marginTop: "calc(-1*var(--sl-spacing-x-small))",
+    },
+    "&[data-steps]  .progress-bar": {
+      marginTop: "var(--sl-spacing-medium)",
+    },
+    "&[data-expired]  .progress::after": {
+      backgroundColor: "#bfbfbf", //"var(--sl-color-neutral-300)",
+    },
+    "&[data-expired]  .filled::after": {
+      backgroundColor: "#bfbfbf", //"var(--sl-color-neutral-300)",
+    },
+    "&[data-complete] .filled::after .progress::after": {
+      backgroundColor: "var(--sl-color-primary-200)",
+    },
+    "& .gift": {
+      textAlign: "center",
+      marginLeft: "-100px",
+      marginRight: "-100px",
+      position: "relative",
+      top: "-18px",
+    },
+    "& .gift.start": {
+      transform: "scale(80%)",
+      top: "-20px",
+    },
+    "& .filled:after": {
+      content: '""',
+      display: "flex",
+      width: "100%",
+      height: "4px",
+      borderRadius: "4px",
+      backgroundColor: "var(--sl-color-primary-300)",
+    },
+    "& .progress": {
+      display: "block",
+      textAlign: "center",
+      marginLeft: "-100px",
+      marginRight: "-100px",
+    },
+    "& .progress::after": {
+      content: '""',
+      width: "12px",
+      height: "12px",
+      display: "flex",
+      backgroundColor: "var(--sl-color-primary-300)",
+      borderRadius: "50%",
+      position: "relative",
+      left: "47%",
+      top: "-85%",
+    },
+    "& .progress.bg:after": {
+      width: "0",
+      height: "0",
+      border: "none",
+    },
+    "& .progress.top": {
+      position: "relative",
+      top: "-40px",
+    },
+    "& .progress.top:after": {
+      top: "-16%",
+    },
+    "& .empty": {
+      display: "block",
+      textAlign: "center",
+      marginLeft: "-100px",
+      marginRight: "-100px",
+    },
+    "& .empty::after": {
+      content: '""',
+      width: "12px",
+      height: "12px",
+      border: "2px solid #E0E0E0",
+      margin: "-2px",
+      display: "flex",
+      backgroundColor: "white",
+      borderRadius: "50%",
+      position: "relative",
+      left: "47%",
+      top: "-85%",
+    },
+    "& .empty.bg:after": {
+      width: "0",
+      height: "0",
+      border: "none",
+    },
+    "& .remain:after": {
+      content: '""',
+      display: "flex",
+      width: "100%",
+      height: "4px",
+      borderRadius: "4px",
+      backgroundColor: "#E0E0E0",
+    },
+  },
+};
+
+const sheet = createStyleSheet(style, { classNamePrefix: "sqm-prog-bar" });
+const styleString = sheet.toString();
+
+export function ProgressBarView(props: ProgressBarProps): VNode {
+  const {
+    progress = 0,
+    goal = 1,
+    steps = false,
+    progressBarUnit = "",
```

Comment:
@kutaycinar Somewhere along the line the progressBarUnit was actually deleted and wasnt caught during QA. This needs to be fixed